src/functions/index.ts

```diff
@@ -3,7 +3,8 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-export * from './a11y/index.ts'
-export * from './emoji/index.ts'
+export * from './a11y/index'
+export * from './dialog/index'
+export * from './emoji/index'
```

Comment:
Still do not like non standard TS with extensions in imports / exports, but :shrug:

Reply:
> non standard TS with extensions

But there is no "standard" in TS saying to omit an extension. This is only a limitation when TS is used `tsc build` as a compiler.

I'd call it an old-school way when TS was used as a language. Since some time (starting `@babel/typescript-preset` to `esbuild` (incl. Vite), `tsx` and TS v5 in general) TS has been moving to be a static code analyze tool for typechecking almost like a linter.

When we use TS only for typecheck/declarations setting corresponding settings in tsconfig, it works fine.

On the other hand, there is a standard actually saying to keep the extension - ECMAScript standard. This is a standard, and it enforces ESM imports to have extensions. This is a requirement for web-browsers. Same for native TS support in runtimes, for example:
- Nodejs: https://nodejs.org/api/typescript.html#determining-module-system
- Deno: https://docs.deno.com/runtime/fundamentals/modules/#importing-modules